UKHomeOffice/sgmr-service
# 1.2.0 SPCR-276 Add a country option to departure and arrival ports

## Description
Add country autocomplete box above the port field on the departure/arrival form

## To Test
- Sign in and create a new voyage form
- You should see a country field on the departure form
- You should see a country field on the arrival form

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
